index.js

```diff
@@ -1 +1,25 @@
 // Code your solution in this file!
+const returnFirstTwoDrivers = (drivers) => {
+    drivers = ['Antonia', 'Nuru', 'Amari', 'Mo'];
```

Comment:
This totally works, but if you declare `drivers` outside of this function, then you won't need to repeat it in the `returnLastTwoDrivers` function. Less duplication is nice!